master/getting-started/bare-metal/bare-metal.md

```diff
@@ -476,14 +472,50 @@ outside the cluster:
    normal host endpoint policy).  In other words, if a host endpoint is defined
    but has no pre-DNAT policies that explicitly allow or deny a particular
    incoming packet, that packet is allowed to continue on its way, and will
-   then be accepted or dropped according to workload policy (if it is going to
-   a local workload) or to normal host endpoint policy (if not).
+   then be accepted or dropped according to normal host endpoint policy applied on
+   forward traffic or a workload policy.
+
+## Policy on forward traffic
+If `applyOnForward` is `false`, host endpoint policy applies to traffic to/from local process only. 
+
+If `applyOnForward` is `true`, host endpoint policy applies to forward traffic include:
+1. Incoming traffic from a host endpoint to a local workload (container/pod/VM).
+2. Outgoing traffic from a local workload to a host endpoint.
+3. Traffic from one host endpoint that is forward to another host endpoint.
```

Comment:
Because the sequence of this list does not matter, you should use bullets instead of numbers.